include/psa/crypto_extra.h

```diff
@@ -1901,6 +1901,342 @@ static inline struct psa_pake_operation_s psa_pake_operation_init( void )
     return( v );
 }
 
+/**
+ * \brief The context for PSA interruptible signing.
+ *
+ * \note  Contents not yet designed as implementation specific.
+ *
+ */
+struct psa_sign_interruptible_context_s
+{
+    size_t num_ops;
```

Comment:
This should probably be private
```suggestion
    size_t MBEDTLS_PRIVATE(num_ops);
```